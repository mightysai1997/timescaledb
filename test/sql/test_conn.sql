SELECT _timescaledb_internal.test_conn();
