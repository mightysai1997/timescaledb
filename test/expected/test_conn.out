SELECT _timescaledb_internal.test_conn();
 test_conn 
-----------
 
(1 row)

